### PR TITLE
Clp/remove sourcechain

### PR DESCRIPTION
**Pull request by Jedi2002**
Removed Sourcechain
Removed Ticker 

Symbol is the only way to work with CLP right now

### CODE REVIEW

File: scripts/bug267.sh [hunk 1]
```diff
@@ -1,16 +1,21 @@
 #!/usr/bin/env bash
 
 
+rm -rf ~/.sifnoded
```
Comment (banshee) [inline scripts/bug267.sh:4]: Should this whole file be deleted?

File: scripts/upgrade.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```
Comment (banshee) [inline scripts/upgrade.sh:1]: This needs some comments or doc.  When do you run it?

File: scripts/bug267.sh (hunk 1 above)
Comment (Jedi2002) [reply scripts/bug267.sh:4]: Yeah , To clean the node .

File: scripts/upgrade.sh (hunk 2 above)
Comment (Jedi2002) [reply scripts/upgrade.sh:1]: https://docs.google.com/document/d/1AqWfSBK8m8XFi23aQ5j3SLhO7AcFWFKG0SWRjUDT3EE/edit

This was more of a research task , I havent added documentation for it yet , the upgrades stuff is nowhere near being finalized . You could still use the above doc for reference

**Comment by utx0** [comment]
Hey @intl-man we are having issue with this PR due to sifgen tool updates that need to happen. Can you please take a look and either help @Jedi2002 with this or if its a simple thing make the needed changes so the intergration tests can pass. Thanks mate.

**Comment by intl-man** [comment]
> Hey @intl-man we are having issue with this PR due to sifgen tool updates that need to happen. Can you please take a look and either help @Jedi2002 with this or if its a simple thing make the needed changes so the intergration tests can pass. Thanks mate.

Done. @Jedi2002 this should be good now, let me know if otherwise.

File: app/app.go [hunk 3]
```diff
@@ -27,6 +29,9 @@ import (
 
 	"github.com/Sifchain/sifnode/x/ethbridge"
 	"github.com/Sifchain/sifnode/x/oracle"
+
+	"github.com/cosmos/cosmos-sdk/x/upgrade"
```
Comment (utx0) [inline app/app.go:33]: Hey mate, looks like you have merged upgrade module stuff into this PR?

**Review by utx0 — CHANGES_REQUESTED** [review]
Hey @Jedi2002 I'm not too sure what has happened here but I am seeing changes for the upgrade/gov module stuff in the `app.go` file. I therefore cant merge this as this needs to be tested separately

**Review by utx0 — APPROVED** [review]
Outside of my minor comment this is great work. We can deal with a clean up of the scripts whenever really.

**Pull request merged**